hardware/DomoticzHardware.h

```diff
@@ -81,7 +81,7 @@ class CDomoticzHardwareBase
 	void SendUVSensor(const int NodeID, const int ChildID, const int BatteryLevel, const float UVI, const std::string &defaultname);
 	void SendBlindSensor(const int NodeID, const int ChildID, const int BatteryLevel, const int Command, const std::string &defaultname);
 	void SendSoundSensor(const int NodeID, const int BatteryLevel, const int sLevel, const std::string &defaultname);
-	void SendAlertSensor(const int NodeID, const int BatteryLevel, const int alertLevel, const std::string &message, const std::string &defaultname);
+	void SendAlertSensor(const int NodeID, const int BatteryLevel, const int alertLevel, const std::string &message, const char * defaultname);
```

Comment:
why going back to char * ? Please don't, you see all sensors have "std::string &defaultname", this one should too